node/src/cli.rs

```diff
@@ -196,6 +199,16 @@ pub struct Cli {
     #[clap(subcommand)]
     pub subcommand: Option<Subcommand>,
 
+    /// Disable automatic hardware benchmarks.
+    ///
+    /// By default these benchmarks are automatically ran at startup and measure
+    /// the CPU speed, the memory bandwidth and the disk speed.
+    ///
+    /// The results are then printed out in the logs, and also sent as part of
+    /// telemetry, if telemetry is enabled.
+    #[clap(long)]
+    pub no_hardware_benchmarks: bool,
```

Comment:
should we use name `hardware_benchmarks` ?

Reply:
Hardware benchmarks should be enabled by default and
```rust
pub no_hardware_benchmarks: bool
```
results in the client having a flag `--no-hardware-benchmarks` which will disable them. Thus I am in favor of keeping it like that.